server.go

```diff
@@ -243,6 +254,16 @@ func (s *server) handleFrontConnections() error {
 		http.Redirect(w, r, "https://tty-share.com", http.StatusMovedPermanently)
 	})
 
+	// Add logging middleware
```

Comment:
@matthewstrasiotto  I know it's been ages since you submitted this, but only now had some time to review. Sorry for the delay, and thanks for the change.

I think this part with the logging should be left out of the trailing slash change, since it's not really related

Reply:
oh lol, fair enough